### PR TITLE
STELLAR-1484 Documentation

### PR DESCRIPTION
**Pull request by reiinakano**

### CODE REVIEW

File: .gitignore
Comment (anuraaga) [inline .gitignore]: Newline

File: README.md
Comment (reiinakano) [inline README.md]: I feel like we need to give more details here but I don't know what our process for requesting API keys is.

File: MANIFEST.md
Comment (vaelen) [inline MANIFEST.md]: Need an email address here.

File: README.md
Comment (anuraaga) [inline README.md]: `[StellarStation]` throughout

File: examples/advanced_usage.grc
Comment (vaelen) [inline examples/advanced_usage.grc]: Do you want to have your home directory name in here?

File: examples/bitstream_source.grc
Comment (vaelen) [inline examples/bitstream_source.grc]: Your home directory again.

File: examples/bitstream_source.grc
Comment (vaelen) [inline examples/bitstream_source.grc]: Home directory

File: MANIFEST.md
Comment (hoshir) [inline MANIFEST.md]: You can omit ", Inc." here.
Also I believe authors@email.address should be replaced to something else. You can use `support@istellar.jp` .

File: examples/iq_source.grc
Comment (vaelen) [inline examples/iq_source.grc]: Home directory

File: examples/iq_source.grc
Comment (vaelen) [inline examples/iq_source.grc]: Home directory

**Review by vaelen — COMMENTED** [review]
You should change "Stellarstation" to "StellarStation".

File: README.md
Comment (hoshir) [reply README.md]: Before we open Amateur API to more people , I think we can just say "An API key can be generated in StellarStation Console."

File: README.md
Comment (hoshir) [inline README.md]: it's safer to not include exact line number in the link until API 1.0.0 since we will have more changes in the API definition.

File: README.md
Comment (reiinakano) [inline README.md]: Btw, I think retrieviing the ID of a particular satellite is not that obvious. Is there a suggested way to do this for external users?

Comment (vaelen) [reply README.md]: The ID will be available in the satellite console.

**Review by vaelen — APPROVED** [review]

File: README.md
Comment (hoshir) [inline README.md]: Do you need `$ cd build` before `cmake ..`?

File: README.md
Comment (hoshir) [inline README.md:15]: Can you specify required major version of Python? (i.e. Python 3 or Python 2)

File: README.md
Comment (hoshir) [inline README.md]: Cmake failed dependency errors of cppuni and Doxygen. 
If preparation is required for the build,  README should comments about it..

Comment (hoshir) [inline README.md]: After installing cppunit and Doxygen, I still have errors:

```
CMake Error at CMakeLists.txt:149 (find_package):
  By not providing "FindGnuradio.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Gnuradio",
  but CMake did not find one.

  Could not find a package configuration file provided by "Gnuradio"
  (requested version 3.7.2) with any of the following names:

    GnuradioConfig.cmake
    gnuradio-config.cmake

  Add the installation prefix of "Gnuradio" to CMAKE_PREFIX_PATH or set
  "Gnuradio_DIR" to a directory containing one of the above files.  If
  "Gnuradio" provides a separate development package or SDK, be sure it has
  been installed.

```

It's OK if users shares knowledge around it.

File: README.md
Comment (hoshir) [inline README.md]: I prefer not including `sudo` in installation command since it is a somewhat risky operation to install something new to their system which might cause damage, and I want to prevent accidental installation by those who don't have much knowledge about what they are doing.  Rather than writing it as a instruction, we can leave it to users by just writing commands without `sudo` with a comment saying "You will need  superuser privilege to execute installation commands."
```
$ sudo make install
$ sudo ldconfig
```

File: README.md
Comment (hoshir) [inline README.md:6]: Can we add target OS of this build? I guess it only work in Linux environment.

Comment (reiinakano) [reply README.md:6]: I don't see a reason why it wouldn't work on any OS running GNURadio, but I haven't tested.

File: README.md
Comment (reiinakano) [reply README.md]: Makes sense.

File: README.md
Comment (reiinakano) [reply README.md]: Did you install GNURadio properly before this? I think your errors are because of improperly installed GNURadio

File: README.md
Comment (reiinakano) [reply README.md:15]: GNURadio only works on Python 2 right now.

File: README.md
Comment (reiinakano) [reply README.md]: Yup, I added this in before merging

File: README.md
Comment (hoshir) [reply README.md:15]: Although Python 2 has extended maintenance period, moving to Python 3 is encouraged. We should consider default version of Python in users are Python 3.x and give extra notice if the build requires older version, Python 2.

File: README.md
Comment (hoshir) [reply README.md]: Yeah, but maybe only binaries are installed by apt-get . Never mind this comment since it looks my environmental issue.

**Pull request merged**
